dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/Assertions.java

```diff
@@ -91,16 +91,32 @@ public static void assertHasOnlyErrors( ValidationReport report, ValidationCode.
         assertHasErrors( report, codes.length, codes );
     }
 
+    /**
+     * assertHasWarnings asserts the report contains only given warnings in any
```

Comment:
Please adapt the javadoc to `assertHasOnlyWarningsAndNoErrors` and its description. Also the overloaded method below.